### PR TITLE
Turning in assignment

### PR DESCRIPTION
**Pull request by calebjwl**

### CODE REVIEW

**Comment by calebjwl** [comment]
* [x] Create a new project named `20-ribbiter` using the `qunit-broccoli` generator
* [x] Run `hub create` to create a new repository on Github
* [x] Push `master` branch
* [x] Create `develop` branch
* [x] **Checkpoint:** Setup Application
  * [x] Install: redux, revue, redux-thunk-rest, and redux-thunk
  * [x] Setup Vue
  * [x] Uncomment `Brocfile.js`, `.eslintrc.js`, and `server/index.js` files
  * [x] Use the `vrouter` snippet to setup the Vue application
  * [x] Create a `store.js` file
  * [x] Create a `reducer.js` file that exports Redux `combineReducers` with a blank object (to be filled later)
  * [x] Create blank components in `routes/application.vue` and `routes/index.vue` so the app doesn't have a build error
* [x] **Checkpoint:** Create an `application` route
  * [x] Add top nav
  * [x] Use a `router-view` for child routes to render
  * [x] Style it!
* [x] **Checkpoint:** Create an `index` route
  * [x] Add a heading that welcomes the user
  * [x] Add buttons that bring the user to `/register` and `/login`
  * [x] Style it!
* [x] **Checkpoint:** Create a `register` route
  * [x] Setup router to respond to `/register`
  * [x] Make HTML for form
  * [x] Style it!
* [x] **Checkpoint:** Create user on `register` form submit
  * [x] Create a file called `resources/user`
  * [x] Use `redux-thunk-rest`'s `createResource` function to create a `user` resource with a `url` to `http://ribbiterapi.herokuapp.com/users` and export the result
  * [x] Setup User Reducer:
    * [x] In `reducer.js` import the user resource
    * [x] For the `users` value in `combineReducers` use the reducer from the user resource (see the `redux-thunk-rest` docs for "Using the reducers")
  * [x] Setup formvalues and `v-model` for various user inputs
  * [x] When the form submits
    * [x] Dispatch `create` from the user resource `actionCreators` passing in data from the form
    * [x] Wait for the dispatch to finish (using `Promise.prototype.then`) and redirect to the `/users` page
  * [x] If there was an error, catch it (using `Promise.prototype.catch`)
    * [x] Using `$select` add `users` to the data for the `register` component
    * [x] Make a `v-if` to check if `users.loading` is equal to `error` and show an error message
    * [ ] The `users` resource will store the error from the server in `users.error`

* [x] **Checkpoint:** Create a `login` page
  * [x] Create `routes/login.vue`
  * [x] Setup router to respond to `/login`
  * [x] Make HTML for form (the functionality of this form will be done in a later assignment)

* [x] **Checkpoint:** Create a `users` page
  * [x] Create `routes/users-index.vue`
  * [x] Setup router to respond to `/users`
  * [x] Create mockup of user cards
* [ ] **CHECKPOINT:** Data for `users` page
  * [x] Dispatch the `findAll` from the `users` resource `actionCreators` when the route is loaded
  * [x] Use `$select` to add the store's `users` data to your local route component
  * [ ] Using `v-for`, loop through the `users.items` and show a card with the `username` and `email` (fake the picture with a picture of your favorite placholder image [square please])

* [x] **Checkpoint:** Create a `ribbits` page
  * [x] Create `routes/ribbits-index.vue`
  * [x] Setup router to respond to `/ribbits`
  * [x] Create mockup of sidebar form
  * [x] Create mockup of list of posts
* [x] **CHECKPOINT:** Resource for `posts`
  * [x] Create a file called `resources/post`
  * [x] Use `redux-thunk-rest`'s `createResource` function to create a `post` resource with a `url` to `http://ribbiterapi.herokuapp.com/posts` and export the result
  * [x] Setup User Reducer:
    * [x] In `reducer.js` import the post resource
    * [x] For the `posts` value in `combineReducers` use the reducer from the post
* [ ] **CHECKPOINT:** Show Data for `ribbits` page
  * [x] Dispatch the `findAll` from the `posts` resource `actionCreators` when the route is loaded
  * [x] Use `$select` to add the store's `posts` data to your local route component
  * [x] Using `v-for`, loop through the `posts.items` and show a post for each item
    * [x] Username
    * [x] Post
* [ ] **CHECKPOINT:** Post Data for `ribbits` page
  * [x] Add formvalues to the `ribbits` component
    * [x] The textarea should fill in the `body` property
  * [x] When the form is submitted
    * [x] Dispatch a `create` actionCreator from the `post` resource
    * [x] Reset formvalues so the form is cleared out

**Comment by calebjwl** [comment]
* [ ] Create `components/gravatar.vue`
* [ ] For the template make an image that has the `url` bound to `https://www.gravatar.com/avatar/${hash}?d=retro&s=200`
* [ ] Add `email` to `props`
* [ ] Make a computed property called `hash` that is the result of passing the email for the current component to `md5`
* [ ] Import the new gravatar component into your `/users` route and use it in place of your placeholder image (don't forget to pass in the `email`)

File: app/routes/users-index.vue [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<template lang="html">
+  <div class="users-index">
+    <div class="section">
+      <div class="users-container">
+        <div class="users-header">
+          <h1 class="users-header__text">Users</h1>
+        </div>
+        <div class="grid">
+          <div v-for="user in users" class="grid__item">
```
Comment (rtablada) [inline app/routes/users-index.vue]: Because of the way that `redux-thunk-rest` stores items, this should loop through `users.items`

**Review by rtablada — COMMENTED** [review]
One quick fix and this will be 👍

**Pull request merged**